### DISCUSSION TITLE
how to list opencv4 default features?

### DISCUSSION THREAD

**Discussion by nocanstillbb** [microsoft/vcpkg#35891] [Q&A]
I got opencv info mation when I linking to opencv debug version
>[ INFO:0@12.879] global D:\Users\user\Documents\source\repos\vcpkg\buildtrees\opencv4\src\4.5.5-923325adf5.clean\modules\core\src\parallel\registry_parallel.impl.hpp (96) cv::parallel::ParallelBackendRegistry::ParallelBackendRegistry core(parallel): Enabled backends(3, sorted by priority): ONETBB(1000); TBB(990); OPENMP(980)
[ INFO:0@12.883] global D:\Users\user\Documents\source\repos\vcpkg\buildtrees\opencv4\src\4.5.5-923325adf5.clean\modules\core\src\utils\plugin_loader.impl.hpp (67) cv::plugin::impl::DynamicLib::libraryLoad load D:\Users\user\Documents\source\repos\dv_app_solution\bin\AMD64\Debug\opencv_core_parallel_onetbb455_64d.dll => FAILED
[ INFO:0@12.926] global D:\Users\user\Documents\source\repos\vcpkg\buildtrees\opencv4\src\4.5.5-923325adf5.clean\modules\core\src\utils\plugin_loader.impl.hpp (67) cv::plugin::impl::DynamicLib::libraryLoad load opencv_core_parallel_onetbb455_64d.dll => FAILED

It seems that opencv `tbb feature` is not installed correctly,
how to list opencv4 default features? 
once i  confirm tbb isn't in default fetaures, i will add it to my vcpkg.json

**Comment by nocanstillbb** [comment]
can i print `${OpenCV_LIB_COMPONENTS}` to comfirm that tbb is a default feature or not?

**Comment by HernandoR** [comment]
any update on this? i'm having trouble identify the default features

**Comment by autoantwort** [comment]
You can scroll here to the default-features feature: https://vcpkg.link/ports/opencv4

**Reply by HernandoR** [reply]
Forgive me for this probably a dummy question, but I truly not really understand the symbols and marks. for instances as following:
![image](https://github.com/user-attachments/assets/4b81a4b5-54de-4cea-af98-91c42964d7f4)
I sincerely hope there is a tutorial or simple explanation for how to read the tags

**Reply by autoantwort** [reply]
The situation now gets simplified with https://github.com/microsoft/vcpkg/pull/43347/files#diff-9ee5e72218274c14ed9a5fd2cfff7c3fa3bf7e01610c58ac47879f1c5037df87L23-L25
